app/controllers/application_controller.rb

```diff
@@ -63,6 +63,11 @@ def clear_intake_session
     session.delete("intake_id")
   end
 
+  def sign_in_and_clear_intake_session(client)
+    sign_in client
+    session.delete("intake_id")
```

Comment:
Should we be even more explicit? Create a new session object retaining just the stuff we know we want?